### PR TITLE
Several updates to everything

### PR DESCRIPTION
**Pull request by txomon**
Main updates:
- Updated for Android 6 (some of us run it already)
- Butterknife for view injection

### CODE REVIEW

File: .idea/.name [hunk 1]
```diff
@@ -1 +1 @@
-AvisaAAne
```
Comment (jonbesga) [inline .idea/.name:1]: Why this change of name?

File: .idea/encodings.xml [hunk 2]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jonbesga) [inline .idea/encodings.xml:1]: Why remove the encoding configurations?

File: .idea/gradle.xml [hunk 3]
```diff
@@ -5,7 +5,7 @@
       <GradleProjectSettings>
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$" />
-        <option name="gradleJvm" value="1.8" />
```
Comment (jonbesga) [inline .idea/gradle.xml:8]: I suppose that this change is because you're using Java 7 instead of the last version, right?

File: app/build.gradle [hunk 4]
```diff
@@ -1,8 +1,8 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 22
```
Comment (jonbesga) [inline app/build.gradle:4]: Why upgrade to 23? What do you need that 22 doesn't have?

File: .idea/misc.xml [hunk 5]
```diff
@@ -1,5 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
+  <component name="EntryPointsManager">
```
Comment (jonbesga) [inline .idea/misc.xml:3]: What are these lines?

File: app/src/main/java/com/besga/jonander/avisaaane/MainActivity.java [hunk 6]
```diff
@@ -1,65 +1,47 @@
...
+            Toast.makeText(getApplicationContext(), "Suscrita a las notificaciones", Toast.LENGTH_SHORT).show();
+        } else {
+            ParsePush.unsubscribeInBackground("Ane");
+            Toast.makeText(getApplicationContext(), "Cancelar suscripcion", Toast.LENGTH_SHORT).show();
+        }
 
+    }
 
+    @OnClick(R.id.btn_notify)
+    public void unsubscribe(View view) {
```
Comment (jonbesga) [inline app/src/main/java/com/besga/jonander/avisaaane/MainActivity.java:40]: Function name doesn't match with the purpose. I'd call it "notify"

File: app/src/main/java/com/besga/jonander/avisaaane/StarterApplication.java [hunk 7]
```diff
@@ -17,18 +20,19 @@ public void onCreate() {
         Properties prop = new Properties();
         try {
             prop.load(getAssets().open("app.properties"));
-
-
         } catch (IOException ex) {
-            ex.printStackTrace();
+            Log.e(TAG, "File app.properties with ApplicationID and ClientKey not found", ex);
+            Intent homeIntent = new Intent(Intent.ACTION_MAIN);
```
Comment (jonbesga) [inline app/src/main/java/com/besga/jonander/avisaaane/StarterApplication.java:25]: Calls the home screen and clears the activities stack, therefore, closing the whole app?

File: .idea/.name (hunk 1 above)
Comment (txomon) [inline .idea/.name:1]: Idea done, I have no idea what it is. everything under .idea/ is not mine

File: .idea/encodings.xml (hunk 2 above)
Comment (txomon) [inline .idea/encodings.xml:1]: Idea done, I have no idea what it is. everything under .idea/ is not mine

File: .idea/gradle.xml (hunk 3 above)
Comment (txomon) [inline .idea/gradle.xml:8]: Idea done, I have no idea what it is. everything under .idea/ is not mine

File: avisa-a-ane.iml [hunk 8]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jonbesga) [inline avisa-a-ane.iml:1]: I don't understand why I didn't have this file generated in my project O_o

File: app/build.gradle (hunk 4 above)
Comment (txomon) [inline app/build.gradle:4]: Support for devices running on api 23

File: .idea/misc.xml (hunk 5 above)
Comment (txomon) [inline .idea/misc.xml:3]: Idea done, I have no idea what it is. everything under .idea/ is not mine

File: app/src/main/java/com/besga/jonander/avisaaane/MainActivity.java (hunk 6 above)
Comment (txomon) [inline app/src/main/java/com/besga/jonander/avisaaane/MainActivity.java:40]: good catch

File: app/src/main/java/com/besga/jonander/avisaaane/StarterApplication.java (hunk 7 above)
Comment (txomon) [inline app/src/main/java/com/besga/jonander/avisaaane/StarterApplication.java:25]: Yep, the idea is that instead of giving no warnings nor anything, it will print the stacktrace of the error and leave. The best would be to create a dialog saying XXX but I was lazy...

File: .idea/gradle.xml (hunk 3 above)
Comment (jonbesga) [inline .idea/gradle.xml:8]: Okay. But check your Java version. We should make sure of we're using the same (to minimize compatibility issues?)
